readme.md

```diff
@@ -28,7 +28,7 @@ Copyright (C) 2021 FreezeFrame
 
 ## Licenses For ImageSharp
 
-- This tool uses [ImageSharp](https://sixlabors.com/products/imagesharp/), which requires a license for commercial use.
+- This tool uses [ImageSharp](https://sixlabors.com/products/imagesharp/), which requires a license for commercial support.
```

Comment:
People can use the libraries commercially under the Apache license. They just can't ask me for any support unless they purchase a Six Labors license.